app/assets/stylesheets/application.scss

```diff
@@ -97,3 +97,49 @@ body {
 .centered {
   text-align: center;
 }
+
+.checkout-step p {
+  margin-top: 10px;
+}
+
+.checkout-row {
+  display: table-row;
+}
+
+.checkout {
+  display: table;
+  position: relative;
+  width: 100%;
+}
+
+.checkout-step button[disabled] {
+  filter: alpha(opacity=100) !important;
+  opacity: 1 !important;
+}
+
+.checkout-row:before {
+  background-color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.